docs/overview.md

```diff
@@ -125,6 +127,14 @@ when they do not otherwise inhibit the primary goals.
   maintained by the backend providers themselves. When that isn't possible,
   custom backends should be able to be configured by end-users with a moderate
   amount of effort.
+- **Native Windows compatibility.** Sprocket is exhaustively tested within a
```

Comment:
Note that, since we ended up supporting it, I have moved native Windows compatibility to goal-adjacent.